packages/profile/src/views/Profile.jsx

```diff
@@ -56,13 +67,46 @@ const reducer = (state, action) => {
 const change = (dispatch, type, e) => {
   try {
     const target = e.target
+
+    if (type === 'avatar') {
+      onAvatarChange(dispatch, target)
+      target.value = null // reset so onChange will fire even for same image
+      return
+    }
+
     const payload = target.type === 'checkbox' ? target.checked : target.value
     dispatch({ type, payload })
   } catch (error) {
     console.log(error)
   }
 }
 
+const onAvatarChange = (dispatch, target) => {
```

Comment:
nit: you should define fns before they are called ex. this is called in change and then defined later. It should be switched.